lib/mail_catcher.rb

```diff
@@ -179,6 +179,14 @@ def run! options=nil
     # Stash them away for later
     @@options = options
 
+    # daemonize mode, reap the child automatically to prevent zombie
+    if options[:daemon] && pid = fork
+      Process.detach(pid)
+      # wait a second to make sure that all output is send before exit
+      sleep 2
```

Comment:
If not sleeping a second the output from command line is printed too fast so that the output of starting mailcatcher will follow later. That confuses the user.